pkg/controller/controller.go

```diff
@@ -644,6 +595,22 @@ func parsePod(obj interface{}) *v1.Pod {
 	return pod
 }
 
+func (ctrl *resizeController) patchPersistentVolume(oldPV, newPV *v1.PersistentVolume) (*v1.PersistentVolume, error) {
+	patchBytes, err := util.GetPatchData(oldPV, newPV)
+	if err != nil {
+		return nil, fmt.Errorf("can't update capacity of PV %s as generate path data failed: %v", newPV.Name, err)
+	}
+	updatedPV, updateErr := ctrl.kubeClient.CoreV1().PersistentVolumes().Patch(context.TODO(), newPV.Name, types.StrategicMergePatchType, patchBytes, metav1.PatchOptions{})
+	if updateErr != nil {
+		return nil, fmt.Errorf("update capacity of PV %s failed: %v", newPV.Name, updateErr)
+	}
+	err = ctrl.volumes.Update(updatedPV)
```

Comment:
If we remove this update and use the common one, the unit tests for resizer started failing

Reply:
Yeah, I don't think we should update the resize controller by removing the cache store's for now. Lets do that in a follow up.